tests/contract_testcases/solana/functions/selectors-must-be-different.sol

```diff
@@ -10,23 +10,23 @@ contract d {
 	int public c;
 
 	// error: selector is the same as c
-	function f1() selector=hex"c3da42b8" public {}
+	function f1() selector=hex"13fbd725feff6e10" public {}
```

Comment:
Since the IDL does not specify selectors/discriminators yet, so how can we support overriding selectors on Solana? This would make contracts which no longer work, unless I'm mistaken.

Reply:
That's a good point. Should we disable overriding selectors on Solana?

Reply:
I added a warning about this.